### DISCUSSION TITLE
error with "Generate transparent video with subtitles"

### DISCUSSION THREAD

**Discussion by hellobigbean** [SubtitleEdit/subtitleedit#8751] [Q&A]
![image](https://github.com/user-attachments/assets/15ab360b-52f6-4419-b200-8eda3c12ccf7)

error with "Generate transparent video with subtitles"

**Comment by niksedk** [comment]
Thx :)
I've tried to prevent 0x0 from happening in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.7/SubtitleEditBeta.zip

**Comment by hellobigbean** [comment]
Fantastic ，thanks ！  @niksedk 
![image](https://github.com/user-attachments/assets/88df2242-8019-4487-a571-9c2ff55c7b0c)
